stdlib/src/builtin/float_literal.mojo

```diff
@@ -275,6 +275,7 @@ struct FloatLiteral(
             return self
 
         alias one = __mlir_attr.`#kgen.int_literal<1> : !kgen.int_literal`
+        alias minus_one = __mlir_attr.`#kgen.int_literal<-1> : !kgen.int_literal`
```

Comment:
nit: to be consistent with neg_zero
```suggestion
        alias neg_one = __mlir_attr.`#kgen.int_literal<-1> : !kgen.int_literal`
```

Reply:
Agreed. Done here: https://github.com/msaelices/mojo/commit/c6cf50f3962561f05d943a5b8016c068e39b6e6f